### ISSUE TITLE
feat: Support $eq query for liveQuery

### ISSUE THREAD

**Issue by sadakchap** [parse-community/parse-server#7606]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation

As per this [issue](https://github.com/parse-community/Parse-SDK-JS/issues/1372), currently it is not possible to combine `equalTo` clause with any other clauses and vice-versa. After adding support for combining `equalTo` clause with other clauses in [Parse-SDK-JS's pr](https://github.com/parse-community/Parse-SDK-JS/pull/1373).

We need to add support `equalTo` for LiveQuery, so that it triggers correct events for client.

### Feature / Enhancement Description

We would need to add support for `$eq` query in [LiveQuery's matchKeyConstraints](https://github.com/parse-community/parse-server/blob/20cb3333aba67fb89e4e2cda586c559bdb4a23ef/src/LiveQuery/QueryTools.js#L148).

### Example Use Case

```js
let q = new Parse.Query('MyClass');
q.equalTo('age', null);
q.exists('age');
// if final query would be like 
// where: { age: { $eq: null, $exists: true } }
const subscribe = await q.subscribe();
```

### Alternatives / Workarounds

---

### 3rd Party References

---

**Comment by cbaker6** [comment]
Looks related to https://github.com/parse-community/parse-server/pull/7227#discussion_r611092794

In the discussion, there are ways to do equalTo in LiveQuery without $eq which are linked in the discussion.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.1.0-alpha.18](https://github.com/parse-community/parse-server/releases/tag/6.1.0-alpha.18)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.3.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/6.3.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.3.0-alpha.1](https://github.com/parse-community/parse-server/releases/tag/6.3.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.3.0](https://github.com/parse-community/parse-server/releases/tag/6.3.0)
